test/PCLExt.FileStorage.NetFX.Test/FilesTest.cs

```diff
@@ -179,7 +224,7 @@ public void OpenUnknown()
         {
             var file = TestFolder.CreateFile(FileName1, CreationCollisionOption.FailIfExists);
 
-            Assert.That(() => file.Open((FileAccess) 2), Throws.ArgumentException);
```

Comment:
You should avoid those formatting changes in a pull request

Reply:
Sorry I didn't get what do you mean exactly. I'll try to figure out

Reply:
(FileAccess) 2 -> (FileAccess)2

Reply:
Got it